server/routerlicious/packages/routerlicious-base/src/test/nexus/io.spec.ts

```diff
@@ -338,130 +342,427 @@ describe("Routerlicious", () => {
 				});
 
```

Comment:
Can we express any easy reliability-related scenarios? What happens if I send bunch of successive Signals, say 100? I just want to know if services crashes, signals get dropped, or some other other reliability issues comes up. Perf is less interesting.

The question above is related to both broadcast and v2 signals.